PublicMapping/districtbuilder
# Update Nest.js deps

## Overview
Update all Nest.js deps, including the "microframework" packages such as `@nestjsx/crud` and `@nestsx/crud-typeorm` and their peer dependencies

### Checklist


### Notes
I tried to make the commit messages nice and descriptive so I'd recommend looking at those one-by-one.

There are some warnings shown when running `yarn install` around unmet peer dependencies, but we don't use those dependencies for templates in email (we're using handlebars) and so they should optional and the warnings can be safely ignored: 
```
warning " > @nestjs-modules/mailer@1.5.1" has unmet peer dependency "@types/ejs@^3.0.3".
warning " > @nestjs-modules/mailer@1.5.1" has unmet peer dependency "@types/pug@2.0.4".
warning " > @nestjs-modules/mailer@1.5.1" has unmet peer dependency "ejs@^3.1.2".
warning " > @nestjs-modules/mailer@1.5.1" has unmet peer dependency "pug@^2.0.4 || ^3.0.0".
```

## Testing Instructions
* Run `scripts/update` and then `scripts/server`
* Click around the app, exercising as much of its functionality as possible to make sure everything still works. I did this:
  * Register new user
  * Log out/log in
  * Create new map
  * Assign districts
  * Share
  * Export CSV 

Closes #602
